sycl/plugins/level_zero/usm_allocator.cpp

```diff
@@ -514,9 +516,24 @@ class USMAllocContext::USMAllocImpl {
   USMAllocImpl(std::unique_ptr<SystemMemory> SystemMemHandle)
       : MemHandle{std::move(SystemMemHandle)} {
 
-    Buckets.reserve(BucketSizes.size());
+    // Buckets for Host and Shared allocations will use the full
+    // set of buckets, starting at the page size of 64 bytes.
```

Comment:
What do you mean by "page size of 64 bytes" ?
What will prevent 2 independent allocations from going into the same cache line (64B)?